contracts/examples/SimpleToken.sol

```diff
@@ -22,7 +22,7 @@ contract SimpleToken is StandardToken {
    * @dev Constructor that gives msg.sender all of existing tokens.
    */
   function SimpleToken() public {
-    totalSupply = INITIAL_SUPPLY;
+    totalSupply_ = INITIAL_SUPPLY;
```

Comment:
I thought we had said that we were going to use an underscore as a prefix, instead of  a sufix, am I right?

Reply:
Yeah, hence the last comment on the PR:
> Regarding the naming convention for the backing field: since we are using leading underscores for the parameters, I unilaterally decided to use trailing underscore for the backing fields. Other options I considered were using a dollar sign ($totalSupply or totalSupply$), or some variant of Hungarian notation (u256TotalSupply).